core/controllers/contributor_dashboard_admin.py

```diff
@@ -529,3 +530,215 @@ def _get_complete_translation_contribution_stats(
             })
 
         return response_translation_contribution_stats_dicts
+
+
+class ContributorDashboardAdminStatsHandlerNormalizedPayloadDict(TypedDict):
+    """Dict representation of ContributorDashboardAdminStatsHandler's
+    normalized_request dictionary.
+    """
+
+    page_size: int
+    offset: int
+    language_code: Optional[str]
+    sort_by: Optional[str]
+    topic_ids: Optional[List[str]]
+    max_days_since_last_activity: Optional[int]
+
+
+class ContributorDashboardAdminStatsHandler(
+    base.BaseHandler[
+        Dict[str, str],
+        ContributorDashboardAdminStatsHandlerNormalizedPayloadDict
+    ]
+):
+    """Return Contributor Admin Dashboard Stats for supplied parameters.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'contribution_type': {
+            'schema': {
+                'type': 'basestring'
+            },
+            'choices': [
+                feconf.CONTRIBUTION_TYPE_TRANSLATION,
+                feconf.CONTRIBUTION_TYPE_QUESTION
+            ]
+        },
+        'contribution_subtype': {
+            'schema': {
+                'type': 'basestring'
+            },
+            'choices': [
+                feconf.CONTRIBUTION_SUBTYPE_SUBMISSION,
+                feconf.CONTRIBUTION_SUBTYPE_REVIEW
+            ]
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'page_size': {
+                'schema': {
+                    'type': 'int'
+                },
+                'validators': [{
+                    'id': 'is_at_least',
+                    'min_value': 0
+                }],
+                'default_value': 20
+            },
+            'offset': {
+                'schema': {
+                    'type': 'int'
+                },
+                'validators': [{
+                    'id': 'is_at_least',
+                    'min_value': 0
+                }]
+            },
+            'language_code': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None,
+                'validators': [{
```

Comment:
I was expecting this to be inside `schema`, can you please check whether it's working here?

@Nik-09 Can we add validators here? (OR do we need to it inside the schema)? If both, I would suggest sticking it to one place i.e, inside the schema.

Reply:
Yes we need to add **choices** inside the **schema**, for ref: https://github.com/oppia/oppia/blob/3d1d9af3d08d8448af866cbe5e5779e4bd9feb90/core/schema_utils.py#L278

Reply:
Okay. Updated it. PTAL. Thanks.